src/pattern-library/components/patterns/IconFoundations.js

```diff
@@ -15,13 +15,13 @@ export default function IconFoundations() {
   return (
     <Library.Page title="Icons">
       <Library.Pattern title="Hypothesis icon set">
-        <div className="LibraryGrid">
+        <div className="grid grid-cols-8 gap-6">
           {Object.keys(icons).map(iconName => (
             <Card
               key={iconName}
-              classes="hyp-u-vertical-spacing hyp-u-layout-column--center IconTile"
+              classes="flex flex-col items-center bg-grey-0 border-[#9c9cab] space-y-3 rounded-lg"
```

Comment:
The border-color here is shortly to be enshrined as an actual color in our palette; it just hasn't happened yet. I'm going to make a note to myself to clean this up ASAP.